src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/ButtonTests/UnoSamples_Test_Button_Click.cs

```diff
@@ -89,5 +90,134 @@ void CheckClickCounts(int layer1Count, int layer1InnerCount, int layer2Count, in
+
+			_app.Wait(0.45f);
+
+			var btnTapped = _app.Marked("btnTapped");
+			var btnTappedRect = btnTapped.FirstResult().Rect;
+			var btnDoubleTapped = _app.Marked("btnDoubleTapped");
+			var btnDoubleTappedRect = btnDoubleTapped.FirstResult().Rect;
+			var btnClick = _app.Marked("btnClick");
+			var btnClickRect = btnClick.FirstResult().Rect;
+			var btnPointerPressed = _app.Marked("btnPointerPressed");
+			var btnPointerPressedRect = btnPointerPressed.FirstResult().Rect;
+			var btnPointerReleased = _app.Marked("btnPointerReleased");
+			var btnPointerReleasedRect = btnPointerReleased.FirstResult().Rect;
+			var btnPointerEntered = _app.Marked("btnPointerEntered");
+			var btnPointerEnteredRect = btnPointerEntered.FirstResult().Rect;
+			var btnPointerExited = _app.Marked("btnPointerExited");
+			var btnPointerExitedRect = btnPointerExited.FirstResult().Rect;
+
+			var btnTappedCount = _app.Marked("btnTappedCount");
+			var btnDoubleTappedCount = _app.Marked("btnDoubleTappedCount");
+			var btnClickCount = _app.Marked("btnClickCount");
+			var btnPointerPressedCount = _app.Marked("btnPointerPressedCount");
+			var btnPointerReleasedCount = _app.Marked("btnPointerReleasedCount");
+			var btnPointerEnteredCount = _app.Marked("btnPointerEnteredCount");
+			var btnPointerExitedCount = _app.Marked("btnPointerExitedCount");
+
+			var hitInvisibleZoneRect = _app.Marked("hitInvisibleZone").FirstResult().Rect;
+			var hitVisibleZoneRect = _app.Marked("hitVisibleZone").FirstResult().Rect;
+
+			var xDirect = (hitInvisibleZoneRect.X - btnTappedRect.X) / 2 + btnTappedRect.X;
+			var xInvisible = hitInvisibleZoneRect.CenterX;
+			var xVisible = hitVisibleZoneRect.CenterX;
+
+			void CheckCount(QueryEx mark, int expected, string msg)
+			{
+				_app.Wait(0.15f); // give time to app to execute the action
+
+				var countString = mark.GetDependencyPropertyValue("Text") as string;
+				var count = int.TryParse(countString, out var c) ? c : -1;
+				count.Should().Be(expected, msg);
+			}
+
+			// ---- TAPPED ----
+			CheckCount(btnTappedCount, 0, "tapped starting value");
+			// Tapped - Direct
+			_app.TapCoordinates(
+				x: xDirect,
+				y: btnTappedRect.CenterY
+				);
+			CheckCount(btnTappedCount, 1, msg: "after direct tapped");
+			// Tapped - Invisible
+			_app.TapCoordinates(
+				x: xInvisible,
+				y: btnTappedRect.CenterY
+			);
+			CheckCount(btnTappedCount, 2, msg: "after tapped though invisible");
+			// Tapped - Visible
+			_app.TapCoordinates(
+				x: xVisible,
+				y: btnTappedRect.CenterY
+			);
+			CheckCount(btnTappedCount, 2, msg: "after tapped though visible");
+			// Tapped - Dragged
+			_app.DragCoordinates(
+				fromX: xVisible,
+				fromY: btnTappedRect.CenterY,
+				toX: btnTappedRect.Right - 1,
+				toY: btnTappedRect.CenterY
+			);
+			CheckCount(btnTappedCount, 2, msg: "after dragged tapped (should not tap)");
+
+			// ---- DOUBLE TAPPED ----
+			/*CheckCount(btnDoubleTappedCount, 0, "double tapped starting value");
```

Comment:
Is there an issue for this disabled test ?

Reply:
Yes...  double tap not supported yet by all platforms on Uno.

Reply:
I hope this code will be uncommented by David's PR on touches...

Reply:
#1256